bin/impl/fetch.sh

```diff
@@ -118,7 +118,7 @@ metrics)
   rm -rf "$GF_PATH"
   ;;
 *)
-  echo "Usage: uno fetch <dependency>"
+  echo "Usage: uno fetch <dependency1> [<dependency2>]"
```

Comment:
I looked for a standard shell syntax for one or more arguments.  It seems using ellipses is a standard way to do this.  That would look like `uno fetch <dependency>...` and would indicate 1 or more deps.  Below is what I found during my search.

 * In the process of searching I found [docopt](http://docopt.org/), which looks like a neat project. Docopt uses ellipses for one or more args.    
 * See 12.1.9 at http://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap12.html
 * After finding above I looked at help for tar and grep.  Both use `[FILE]...` which indicates zero or more files.